climada/engine/impact_calc.py

```diff
@@ -129,6 +129,19 @@ def impact(self, save_mat=True, assign_centroids=True,
                 )
 
         impf_col = self.exposures.get_impf_column(self.hazard.haz_type)
+
+        # check for compability of impact function id between impact function set and exposure
+        if any(if_id not in self.impfset.get_ids(haz_type=self.hazard.haz_type) for
```

Comment:
Please use the similar variable names as in the rest of the file: `if` -> `impf`. 

Besides, a variable name `if` is rather confusing as this is also a python operator. Note that we had a longer discussion on that 2 years ago when we decided to avoid the variable name `if`.

Reply:
Thanks for the notice, I changed `if_id `to `impf_id`.